apps/www/src/app/global-error.tsx

```
@@ -0,0 +1,32 @@
+"use client";
+
+import * as React from "react";
+import Error from "next/error";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)